modules/standard/FileSystem.chpl

```diff
@@ -427,6 +427,29 @@ proc sameFile(file1: file, file2: file): bool {
   return result;
 }
 
+/* Create a symbolic link pointing to actual named sym.  May generate an error
+   message.
+   err: a syserr used to indicate if an error occurred during creation
+   actual: the source file to be linked
+   symLink: the location the symbolic link should live
+*/
+proc symlink(out err: syserr, actual: string, symLink: string) {
```

Comment:
The symlink api (in C, etc) has always been confusing, due to the argument names and their order. Other languages have used different argument names in order to make this api easier to work with. For examples, go and ruby use oldname and newname. Java uses link and target. Rust uses src and dst.

actual and symLink doesn't help me understand which side is which. What do you think about these (just brainstorming):

* `destName` and `symLinkName`
* `oldName` and `newName`
* `target` and `symLink`
* `destination` and `source`

?